content/developers/metrics/distributions.md

```diff
@@ -10,6 +10,10 @@ further_reading:
   text: Official and Community-contributed API and DogStatsD client libraries
 ---
 
+<div class="alert alert-warning">
+    This is documentation for a beta feature.  Please <a href="https://docs.datadoghq.com/help/">contact support</a> to enable distribution metrics for your account.
```

Comment:
Remove leading whitespace. Single space between sentences.

Reply:
Pushed an update that also fixes the source I copy/pasted from.